api/Selection.json

```diff
@@ -69,7 +69,7 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "9"
```

Comment:
https://caniuse.com/selection-api agrees with this. There was something in IE6-8, but not `document.getSelection()` or `window.getSelection()`, so this seems right.